libs/sdk-ui-ext/src/insightView/InsightTitle.tsx

```diff
@@ -0,0 +1,26 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import Truncate from "react-truncate";
+/**
+ * @Public
+ */
+export interface IInsightTitle {
```

Comment:
Please use the `Props` suffix for props types, so here `IInsightTitleProps`

Reply:
Suffix addded.